draft-ietf-privacypass-auth-scheme.md

```diff
@@ -197,6 +197,13 @@ WWW-Authenticate: PrivateToken challenge=abc..., token-key=123...,
 PrivateToken challenge=def..., token-key=234...
 ~~~
 
+If a Client fetches a batch of multiple tokens for future use that are bound
+to a specific redemption context (the redemption_context in the TokenChallenge
+was not empty), Clients SHOULD discard these tokens upon flushing state such as
+HTTP cookies {{?COOKIES=I-D.ietf-httpbis-rfc6265bis}}, or changing networks.
+Using these tokens in a context that otherwise would not be linkable to the 
+original context could allow the Origin to recognize a Client.
+
```

Comment:
```suggestion
Using these tokens in a context that otherwise would not be linkable to the
original context could allow the Origin to recognize a Client.

```